apps/www/lib/redirects.js

```diff
@@ -1565,4 +1565,9 @@ module.exports = [
     source: '/docs/guides/functions/examples',
     destination: '/docs/guides/functions',
   },
+  {
+    permanent: false,
```

Comment:
Should be true (which is a 301 permanent redirect).

Reference:
https://supabase.slack.com/archives/C0147EPF36X/p1655156671001669?thread_ts=1654596769.943899&cid=C0147EPF36X